_docs/layout/page-layouts.md

```diff
@@ -16,35 +16,27 @@ resource: true
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
 ### Base layouts
-- Use the account navigation bar to navigate between applications,
-  manage roles, and manage preferences on both the user and the account
-  level.
+- Use the 'Utility nav' to navigate between applications, manage roles, and manage preferences on both the user and the account level.
```

Comment:
Revise: "Use the Utility navigation (nav)..." This way we establish that nav is short for navigation.